builtin/logical/cassandra/path_roles.go

```diff
@@ -123,11 +128,14 @@ func (b *backend) pathRoleCreate(
 		return logical.ErrorResponse(fmt.Sprintf(
 			"Error parsing lease value of %s: %s", leaseRaw, err)), nil
 	}
+	
+	consistencyStr := data.Get("consistency").(string)	
```

Comment:
Will do. It makes more sense to catch an issue with the consistency level before the role is stored.